src/main/java/seedu/recipe/logic/parser/AddStepCommandParser.java

```diff
@@ -31,12 +31,7 @@ public AddStepCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_STEP);
         List<Step> newSteps = parseStepsForAddStep(argMultimap.getAllValues(PREFIX_STEP));
 
-        Index index;
-        try {
-            index = ParserUtil.parseIndex(argMultimap.getPreamble());
-        } catch (ParseException pe) {
-            throw new ParseException(ParserUtil.MESSAGE_INVALID_INDEX);
-        }
+        Index index = ParserUtil.parseIndex(argMultimap.getPreamble());
```

Comment:
May I ask why you would like to remove this try-catch block?

Reply:
for the same reason that you pointed out last time! :) Right now, the parse exception that is caught is `ParserUtil.MESSAGE_INVALID_INDEX` so I don't have to catch it and rethrow the same error message since the `execute` method already throws ParseException.

Reply:
Alright! thanks!!